examples/dynamic_api/storageclass-wekafs-dir-api.yaml

```diff
@@ -9,8 +9,18 @@ allowVolumeExpansion: true
 parameters:
   volumeType: dir/v1
   filesystemName: default
-  # if the values below are not set, quota will not be enforced for this storage class
 
+  # optional parameters setting UID, GID and permissions on volume
+  # UID of the volume owner, default 0 (root)
+  #ownerUid: "1000"
+
+  # GID of the volume owner, default 0 (root)
+  ownerGid: "1000"
```

Comment:
now it is inconsistent... either all permissions related should present or just one
I prefer all of them commented out, which is standard approach

Reply:
have no idea how I missed it.